uint8 module_id
bool  turn_on
---
